### PR TITLE
Fixed the social media icons

### PR DESCRIPTION
**Pull request by subhahu123**
# Changes done in this Pull Request

Added all.css in social icons (div)
adding it at the top is overiting the other class
for now website looks bad because of some icons using the class of all.css 
for now i m adding the css in the specific class

- Fixes #497 

### Before:
https://user-images.githubusercontent.com/13624560/48975726-7cda1980-f09d-11e8-9b7a-205e8ae64838.png
### After:
https://user-images.githubusercontent.com/13624560/48975723-6df36700-f09d-11e8-94f6-91aa69daa0db.png

### CODE REVIEW

**Comment by subhahu123** [comment]
@jogendra  now it's fine

File: index.html [hunk 1]
```diff
@@ -180,8 +180,8 @@ <h1>Team</h1>
       <div class="col-md-8 col-md-offset-2 text-center">
         <h1>Connect with FOSSASIA on Social Media</h1>
         <p>Be sure to follow us on social media!</p>
-        <div id="social_icons">
-            <a href="https://www.facebook.com/fossasia/" class="ti-facebook lgi" target="_blank" rel="noopener noreferrer">
+        <div id="social_icons">
```
Comment (abishekvashok) [inline index.html]: There are some trailing whitespaces

File: index.html [hunk 2]
```diff
@@ -180,8 +180,8 @@ <h1>Team</h1>
       <div class="col-md-8 col-md-offset-2 text-center">
         <h1>Connect with FOSSASIA on Social Media</h1>
         <p>Be sure to follow us on social media!</p>
-        <div id="social_icons">
-            <a href="https://www.facebook.com/fossasia/" class="ti-facebook lgi" target="_blank" rel="noopener noreferrer">
+        <div id="social_icons">   
+          <a href="https://www.facebook.com/fossasia/" class="ti-facebook lgi" target="_blank" rel="noopener noreferrer">
```
Comment (abishekvashok) [inline index.html]: Please intend this line correctly

**Comment by abishekvashok** [comment]
Also, can you resolve the conflicts as well?

File: index.html (hunk 2 above)
Comment (subhahu123) [reply index.html]: done

File: index.html (hunk 1 above)
Comment (subhahu123) [reply index.html]: done

**Comment by subhahu123** [comment]
> Also, can you resolve the conflicts as well?

i tried to resolve conflict but it is showing no conflicts in gti and gitkoren
i have tried both cli and gui

**Comment by abishekvashok** [comment]
Pull from upstream

**Comment by subhahu123** [comment]
now it's done 
merge it

**Pull request closed without merging**